executor/models/curvenet/curvenet_utils.py

```diff
@@ -0,0 +1,508 @@
+"""Adapted from: https://github.com/tiangexiang/CurveNet/blob/main/core/models/curvenet_util.py
+"""
+from time import time
+from tracemalloc import start
+
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+from .walk import Walk
+
```

Comment:
I prefer to put these functions: `knn`/`normal_knn` ... to a `utils` folder if they can be used by other models.

Reply:
Till now each model has its own logic and utilized functions, their authors may modify the functions to adapt their models. So I think now it's not necessary to summary a `utils` folder for all models.